src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/AbstractReferenceFinder.cs

```diff
@@ -18,6 +18,9 @@ namespace Microsoft.CodeAnalysis.FindSymbols.Finders
 {
     internal abstract partial class AbstractReferenceFinder : IReferenceFinder
     {
+        public const string s_containingTypeInfo = "ContainingTypeInfo";
```

Comment:
Nit: non-private constant fields should start with a capital letter, say `ContainingTypeInfoPropertyName`